l5kit/l5kit/visualization/utils.py

```diff
@@ -91,3 +92,23 @@ def draw_reference_trajectory(on_image: np.ndarray, world_to_pixel: np.ndarray,
     positions_in_pixel_space = positions_in_pixel_space[mask]
     for pos in positions_in_pixel_space:
         cv2.circle(on_image, tuple(np.floor(pos).astype(np.int32)), 1, REFERENCE_TRAJ_COLOR, -1)
+
+
+def draw_path_prior_layer(shape: Tuple, positions: np.ndarray, width: float = 3) -> np.ndarray:
```

Comment:
Consider adding an argument to enable and disable varying the length

Reply:
Thanks, added.